support/sl-post-hook.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
Yeah, it's a bit funky. I originally added this because the Debian image doesn't have `sudo` which the chef-client install uses even as root. Then I added `curl` which also isn't installed by default so that I could use curl and userMetadata to ensure that this script was done before letting chef-provisioning-fog think the server was "good to go".